src/test/java/com/griddynamics/qa/CalculatorTest.java

```diff
@@ -0,0 +1,44 @@
+package com.griddynamics.qa;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.allure.annotations.Features;
+import ru.yandex.qatools.allure.annotations.Stories;
+
+import java.lang.reflect.Method;
+
+
+@Features("Calculator testing")
+public class CalculatorTest extends AbstractTest {
+    @DataProvider
+    public Object[][] DataProvider(Method m) {
+        return new Object[][]{new Object[]
+                {4, 5},
+                {0, 0},
+                {5, -5},
+        };
+    }
+
+    @Test(dataProvider = "DataProvider", groups = "add")
+    @Stories("Add test")
+    public void addTest(int a, int b) {
+        System.out.println("Add test with values (" + a + ", " + b + ")");
+        Assert.assertEquals(a + b, Calculator.add(a, b));
```

Comment:
Would you mind to add a meaningful assertion message? (same for the like #35, #42).